packages/server/src/api/roster/roster.controller.ts

```diff
@@ -320,6 +320,17 @@ class RosterController {
     res.json(entry);
   }
 
+  async patchRosterEntry(req: ApiRequest<OrgRosterParams, RosterEntryData>, res: Response) {
+    assertRequestParams(req, ['rowId']);
+    const entryId = +req.params.rowId;
```

Comment:
We may want to do a `parseInt` here and then check to make sure it's not NaN. Otherwise it looks like we'll get a SQL error.

Reply:
Yup!